addons/mail/static/src/core/common/store_service.js

```diff
@@ -148,6 +148,7 @@ export class Store extends BaseStore {
             description: _t("Notify everyone"),
         },
     ];
+    markdownLoaded = false;
```

Comment:
It's strange to have a loaded state in a different file than the load method, they should be together.

Reply:
I understand, but this library should not be loaded for each message, so it's marked loaded once for all models.

Reply:
The load function should be in store not message

Reply:
I moved everything inside a store function